### ISSUE TITLE
Required field of a Group field blocks submission when not showing

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#784]
**Description**
The field is like it's set as required even if the condition don't match, and that the field is not shown.

**Steps to reproduce**
1. Setup a multi-pages form
2. Setup a Group field
3. Setup a Radio Buttons field
4. Setup a required Date field
5. Add condition to only show the Date field if one option choice of the Radio Buttons field is selected
6. Make a new submission, and select the one of the Radio Buttons that doesn't show the required Date field
7. Try to Submit: you're redirect to the same page tab
8. Check the log: it says that the required hidden field is missing a value

### Form settings
- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.10
- Craft version: Craft Pro 3.7.34
- Multi-site: Yes

**Comment by engram-design** [comment]
Looks like this was an issue with client-side validation for Group fields and conditionally hidden nested fields. Client-side validation works as expected.

Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.11](https://github.com/verbb/formie/releases/tag/1.5.11)
